doxia-modules/doxia-module-markdown/src/test/resources/test.md

```diff
@@ -2,3 +2,12 @@ Markdown test document
 ======================
 
 Placeholder for future expansion.
+
+- item1
+- item2
+- item3
+    - item31
```

Comment:
Isn't this too much indentation?

Reply:
https://docs.newrelic.com/docs/style-guide/structure/styleguide-markup-indentation/#two-space

Reply:
4 spaces is most compatible with different markdown implementations and works under all circumstances.